### PR TITLE
Add Users to Admins and Readers

### PR DESCRIPTION
**Pull request by Richard87**

### CODE REVIEW

File: pkg/apis/radixvalidators/validate_rr_test.go [hunk 1]
```diff
@@ -56,7 +56,6 @@ func TestCanRadixApplicationBeInserted(t *testing.T) {
 		{"empty ad group", func(rr *v1.RadixRegistration) { rr.Spec.AdGroups = []string{""} }, nil},
 		{"empty configBranch", func(rr *v1.RadixRegistration) { rr.Spec.ConfigBranch = "" }, nil},
 		{"invalid configBranch", func(rr *v1.RadixRegistration) { rr.Spec.ConfigBranch = "main.." }, nil},
-		{"empty ad groups", func(rr *v1.RadixRegistration) { rr.Spec.AdGroups = nil }, []radixvalidators.RadixRegistrationValidator{radixvalidators.RequireAdGroups}},
```
Comment (nilsgstrabo) [inline pkg/apis/radixvalidators/validate_rr_test.go:59]: Add back test to check that AD groups cannoty be empty

File: pkg/apis/radixvalidators/validate_rr_test.go [hunk 2]
```diff
@@ -114,7 +113,6 @@ func TestCanRadixApplicationBeUpdated(t *testing.T) {
...
```
Comment (nilsgstrabo) [inline pkg/apis/radixvalidators/validate_rr_test.go:117]: Add back test

**Review by nilsgstrabo — APPROVED** [review]
